### PR TITLE
fusion eltwise to mvn and fusion test

### PR DESCRIPTION
**Pull request by chenhu-wang**
ticket#: 46191

### CODE REVIEW

File: ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp [hunk 1]
```diff
@@ -42,6 +43,19 @@ namespace ngraph
                     out[i] =
                         arg[i] < T(0) ? T(arg[i] * slope[cnt++ % shape_size(slope_shape)]) : arg[i];
                 }
+                // assert(arg_shape.size() > 2);
```
Comment (ilyachur) [inline ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp:46]: Why did you add a commented code?

Comment (chenhu-wang) [reply ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp]: @ilyachur  I removed the below commented code. 
`assert(arg_shape.size() > 2);
size_t batch_size = arg_shape[0] * arg_shape[1];
size_t sptial_size = shape_size(arg_shape) / batch_size;
for (int n = 0; n < batch_size; ++n)
{
    T slope_n = shape_size(slope_shape) == 1 ? slope[0] : slope[n % shape_size(slope_shape)];
    for (int sp = 0; sp < sptial_size; ++sp)
    {
        size_t inx = n * sptial_size + sp;
        out[inx] =
            arg[inx] < T(0) ? T(arg[inx] * slope_n) : arg[inx];
    }
}`

This commented code is just for temporal test and communication. When I test prelu as fused post ops in cpu plugin, with slope of size C(channel size), each channel share the same slope. e.g. data[2,8,5,5], slope[1,8,1,1]. Commented code assuming data is planar layout in ngraph and have same result as in cpu plugin. So do we need some broadcast here for prelu or broadcast was done already in other place? please correct me if I am misunderstanding something.

**Comment by chenhu-wang** [comment]
@dmitry-gorokhov could you please review?

File: ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp (hunk 1 above)
Comment (chenhu-wang) [reply ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp]: @ilyachur I added shape broadcast feature for prelu according [ops specification](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/PReLU_1.md), could you please take a look for ngraph part? Thank you.

File: ngraph/core/src/op/prelu.cpp [hunk 2]
```diff
@@ -135,5 +140,6 @@ namespace prelu
 bool op::PRelu::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v0_PRelu_evaluate);
-    return prelu::evaluate_prelu(inputs[0], inputs[1], outputs[0]);
+    AutoBroadcastSpec broadcast_spec = AutoBroadcastSpec("NUMPY");
```
Comment (ilyachur) [inline ngraph/core/src/op/prelu.cpp:143]: As I know and see in your code the PReLU operation doesn't have and support broadcast.
Why did you add a broadcast support to the reference implementation? Here as I see you also always hardcode broadcast to "NUMPY".

Comment (chenhu-wang) [reply ngraph/core/src/op/prelu.cpp]: Extended, default is "NUMPY". Specified broadcast type can be passed in. As multiple inherit introduce lot of ambiguity, I incorporate broadcast function into op::PRelu directly. Thank you!

Comment (ilyachur) [reply ngraph/core/src/op/prelu.cpp]: @chenhu-wang We cannot extend existed nGraph operations. I propose to introduce the new version of PReLU operation with broadcast support if it is needed.
@lazarevevgeny Do you agree with me?

Comment (lazarevevgeny) [reply ngraph/core/src/op/prelu.cpp]: @ilyachur is right. We cannot modify existing ngraph operations. We must add a new version of the operation and modify all the transformations which use v1::PRelu operation to work with a new version of the operation. Moreover, we need to modify the MO to emit a new version of the operation because right now it emits v1::PRelu.

Comment (chenhu-wang) [reply ngraph/core/src/op/prelu.cpp]: @ilyachur @lazarevevgeny ,I agree we should support backward compatibility, but from PRelu specification: https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/PReLU_1.md, saying broadcasted capability is supported for this version. Should we modify the old one and add an new version spec for prelu?

Comment (lazarevevgeny) [reply ngraph/core/src/op/prelu.cpp]: We should update the v1 specification that only the numpy style broadcasting is supported and create a new document for a v7 version of the PReLU (if we plan to introduce it in the upcoming release).

Comment (chenhu-wang) [reply ngraph/core/src/op/prelu.cpp]: this changes should be in other PR and reverted here, thank you.

**Review by lazarevevgeny — APPROVED** [review]
Now the PR does not contain ngraph-related changed. Setting approve.

**Comment by dmitry-gorokhov** [comment]
@chenhu-wang Thanks for big effort while implemented this fusing approach for MVN node. Unfortunately, I am not sure this is how we should actually do it. My main concerns is that current implementation was done by EltwiseNode example and as a results contains huge amount of copypaste (and a lot of copied limitations). Given that it doesn't look like a good idea to spread the approach through all other layers which support post ops.

Actually the ticket 46191 says that we need to support MVN + ELU fusion. Ass I can see old implementation already has necessary code: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp#L578. That's why all you need to do is to extend fusing conditions: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp#L1423 to support all activations and update tests.

We are planning to revise existing post ops handling mechanism and to find better approach that can be applied for all plug-in nodes. Lets discuss offline if you have any concerns.

**Comment by chenhu-wang** [comment]
see https://github.com/openvinotoolkit/openvino/pull/5272
